contracts/staking/fuse/temp/StakingRewardsPerEpoch.sol

```diff
@@ -0,0 +1,207 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import "./StakingRewards.sol";
+import "./GoodDollarSwaps.sol";
+import "./IConsensus.sol";
+
+contract StakingRewardsPerEpoch is StakingRewards, GoodDollarSwaps {
+  using SafeERC20 for IERC20;
+
+  struct StakeInfoPerEpoch {
+    uint256 pendingStake;
```

Comment:
is that the right name???? we already talked about it

Reply:
The conversation in the slack.

Reply:
please  write the decision  of the slack conversations and implement them